Makefile

```diff
@@ -103,7 +103,7 @@ lcov:
 
 apidoc:
 	mkdir -p apidoc
-	doxygen doxygen/mbedtls.doxyfile
+	cd doxygen; doxygen mbedtls.doxyfile
```

Comment:
Change `cd doxygen;` to `cd doxygen &&` so that the second command is not executed if the first fails. This works both under Unix/Posix/Linux and under Windows